sockeye/data_io.py

```diff
@@ -709,14 +736,14 @@ def get_training_data_iters(source: str, target: str,
     """
     Returns data iterators for training and validation data.
 
-    :param source: Path to source training data.
+    :param sources: Path to source training data (with optional factor data paths).
     :param target: Path to target training data.
-    :param validation_source: Path to source validation data.
+    :param validation_sources: Path to source validation data. (with optional factor data paths).
```

Comment:
Remove the period before the opening parenthesis.

Reply:
fixed